neuralprophet/forecaster.py

```diff
@@ -705,6 +705,9 @@ def fit(self, df, freq="auto", validation_df=None, progress="bar", minimal=False
         # When only one time series is input, self.id_list = ['__df__']
         self.num_trends_modelled = len(self.id_list) if self.config_trend.trend_global_local == "local" else 1
         self.num_seasonalities_modelled = len(self.id_list) if self.config_season.global_local == "local" else 1
+        self.meta_name_bool = (
+            False if (self.num_trends_modelled == 1 and self.num_seasonalities_modelled == 1) else True
```

Comment:
```suggestion
            self.num_trends_modelled != 1 or self.num_seasonalities_modelled != 1
```
Should do not the same just easier to read

Reply:
Thank you @noxan !